packages/astro/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@spotlightjs/astro",
+  "description": "Add spotlight to your Astro toolbar, for better debugging.",
+  "version": "0.0.0",
+  "type": "module",
+  "keywors": [
+    "astro",
+    "withastro",
+    "astro-component",
+    "analytics"
```

Comment:
Let's remove the analytics keyword for now. I was tempted to add it in https://github.com/getsentry/sentry-javascript/pull/9193#discussion_r1356768133 but held off from it because the Astro folks said they'd add a better category. I think something like "devtools" would fit better overall. 
```suggestion
```